sktime/transformations/series/scaledlogit.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements the scaled logit tranformation."""
```

Comment:
still typo

Reply:
It was 2 typos! Good attention to detail. 
Fixed it